.semaphore/semaphore.yml

```diff
@@ -90,9 +90,12 @@ blocks:
           - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
       jobs:
         - name: Screener
+          env_vars:
+            - name: SEMAPHORE_BUILD_NUMBER
+              value: $SEMAPHORE_WORKFLOW_ID
```

Comment:
Well, that's not right 😒 

![Screen Shot 2020-01-31 at 2 40 08 PM](https://user-images.githubusercontent.com/14944896/73579599-ac21ae80-4437-11ea-9ea0-997d64237a75.png)

Reply:
i don't know if semaphore.yml allows references to the environment, but you can try `'${SEMAPHORE_WORKFLOW_ID}'`